csv-bulk-signature/src/main.ts

```diff
@@ -255,8 +257,12 @@ async function main() {
   // 5. create report
   await createStatusReport(contactsWithNewProps);
 
-  // 6. zip up files
-  await zipUpFile();
+  // 6. gather files for zip
+  const signatureFolder = await fs.readdir(SIGNATURES_PATH);
+  const files = signatureFolder.filter(filterFilesForZip);
+
+  // 7. zip up files
+  await zipUpFiles(files);
   await checkZipIsNotEmpty(ZIP_FILE);
```

Comment:
gather the files as one step to trim out the multiple test that were in zipUpFiles